core/src/main/scala/org/apache/spark/util/ClosureCleaner.scala

```diff
@@ -159,6 +160,36 @@ private[spark] object ClosureCleaner extends Logging {
     clean(closure, checkSerializable, cleanTransitively, Map.empty)
   }
 
+
+  private def getSerializedLambda(closure: AnyRef): Option[SerializedLambda] = {
+    val isClosureCandidate =
+      closure.getClass.isSynthetic &&
+        closure
+          .getClass
+          .getInterfaces.exists{x: Class[_] => x.getName.equals("scala.Serializable") }
+
+    if (isClosureCandidate) {
+      try {
+        val res = inspect(closure)
+        Some(res)
+      } catch {
+        case e: Exception =>
+          if (log.isDebugEnabled) {
+            logDebug(e.getMessage)
```

Comment:
I agree. I didnt pay much attention here because I only care for failure. I failed (it does not matter why) it is not a LMF. I can print the stacktrace though.